amethyst_core/src/transform/components/local_transform.rs

```diff
@@ -36,22 +36,24 @@ impl Transform {
     ///
     /// # Examples
     ///
-    /// ```rust,no_run
-    /// # use amethyst_core::transform::components::Transform;
-    /// # use amethyst_core::cgmath::{Quaternion, One, Vector3, Point3, Matrix3};
+    /// ```
+    /// # #[macro_use] extern crate cgmath;
+    /// # extern crate amethyst;
+    /// # fn main() {
+    /// use amethyst::core::Transform;
+    /// use amethyst::core::cgmath::{Quaternion, Euler, One, Vector3, Point3, Rad};
+    /// use std::f32::consts::PI;
+    ///
     /// let mut t = Transform::default();
     /// // No rotation by default
     /// assert_eq!(t.rotation, Quaternion::one());
     /// // look up with up pointing backwards
-    /// t.look_at(Point3::new(0.0, 1.0, 0.0), Vector3::new(0.0, 0.0, 1.0));
+    /// t.look_at(Point3::new(0.0, 1.0, 0.0), Vector3::new(1.0, 0.0, 0.0));
```

Comment:
I have the feeling the first one my more sense in respect to the comment saying up is z+

Reply:
Maybe it's best we wait just a little to merge this since Ellie is so far along with nalgebra.. I'd really like to make visualizations for these things to help me and others but mostly me

Reply:
Because I feel as confused as you do here. I developed this current solution (as if this is a puzzle lol) with @Rhuagh